releasenotes/notes/0.10/add-noise-passes-1cb52b57a6d2294d.yaml

```diff
@@ -32,13 +32,13 @@ features:
     :class:`~qiskit.providers.aer.noise.LocalNoisePass` using a
     predefined noise function that returns a tensor product of
     :func:`~qiskit.providers.aer.noise.thermal_relaxation_error` on each
-    qubit in an instruction dependent on the instructions duration and
+    qubit in an instruction, dependent on the instruction's duration and
     the supplied relaxation time constant parameters of the pass.
   - |
     The basic device noise model implemented by
-    :meth:`qiskit.providers.aer.noise.NoiseModel.from_backend` and
-    :meth:`qiskit.providers.aer.AerSimulator.from_backend` has been
-    upgraded to allow adding duration dependent relaxation error on
+    :meth:`.NoiseModel.from_backend` and
```

Comment:
Same here?

Reply:
this displays `NoiseModel.from_backend()` and links correctly, without the noise of the fully qualified path.